docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md

```diff
@@ -0,0 +1,612 @@
+                    "isCollapsedByDefault": false,
+                    "displayProductCount": true,
+                    "categories": {
+                      "edges": [
+                        {
+                          "node": {
+                            "entityId": 19,
+                            "isSelected": false,
+                            "productCount": 1,
+                            "subCategories": {
+                              "edges": []
+                            }
+                          }
+                        },
+                        {
+                          "node": {
+                            "entityId": 21,
+                            "isSelected": false,
+                            "productCount": 1,
+                            "subCategories": {
+                              "edges": []
+                            }
+                          }
+                        }
+                      ]
+                    }
+                  }
+                },
+                {
+                  "node": {
+                    "__typename": "OtherSearchFilter",
+                    "name": "Other",
+                    "isCollapsedByDefault": true,
+                    "displayProductCount": true,
+                    "freeShipping": null,
+                    "isInStock": {
+                      "isSelected": false,
+                      "productCount": 2
+                    },
+                    "isFeatured": null
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  }
+```
+<!-- type: tab-end -->
+
+If a merchant has not enabled product filtering, you will receive an empty array for the returned facets, though products will still be returned:
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example" lineNumbers
+{
+  "data": {
+    "site": {
+      "search": {
+        "searchProducts": {
+          "products": {
+            "edges": [
+              {
+                "node": {
+                  "entityId": 93,
+                  "name": "[Sample] 1 L Le Parfait Jar",
+                  "prices": {
+                    "price": {
+                      "value": 7
+                    }
+                  }
+                }
+              },
+              {
+                "node": {
+                  "entityId": 80,
+                  "name": "[Sample] Orbit Terrarium - Large",
+                  "prices": {
+                    "price": {
+                      "value": 109
+                    }
+                  }
+                }
+              }
+            ]
+          },
+          "filters": {
+            "edges": []
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+<!-- type: tab-end -->
+
+## Resources
+
+- [GraphQL Storefront API Overview](/api-docs/storefront/graphql/graphql-storefront-api-overview)
+- [GraphQL Storefront Playground](/graphql-playground)
```

Comment:
```suggestion
- [GraphQL Storefront Playground](https://developer.bigcommerce.com/graphql-playground)
```
the playground isn't part of the Stoplight docs app, so it needs the whole URL. (this is also makes it open in a new window, which is great IMO for its use case.)